README.md

```diff
@@ -85,13 +85,7 @@ This is a listing of example projects and guides that will help you integrate Se
 
 <details>
   <summary>Show projects</summary>
-
-  ### [webpack][28]
-  See our webpack 3 example project [here][28] (includes theming).
-   
-  ### [SUIcrux][102]
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/88539028-bc929a80-d010-11ea-9a74-fba693336d04.png)

The project is not maintained anymore.